app/build.gradle

```diff
@@ -62,6 +62,7 @@ android {
 
     buildTypes {
         release {
+            buildConfigField "String", "RC_SERVER_URL", '""'
```

Comment:
Move to `defaults.xml` file please. 

string tag can be just `server_url` there.